tests/tf/algos/test_trpo.py

```diff
@@ -22,6 +22,9 @@ def setUp(self):
         self.sess.__enter__()
         logger._tensorboard = TensorBoardOutput()
 
+    def tearDown(self):
```

Comment:
You can make a custom TestCase class (inheriting from `unittest.TestCase`) which overloads setUp/tearDown, and inherit from that (put it in tests/fixtures.py).

Reply:
sounds good